shared/router-v2/router.desktop.js

```diff
@@ -71,7 +71,7 @@ class AppView extends React.PureComponent<any> {
           style={selectedTab ? styles.contentArea : styles.contentAreaLogin}
         >
           {scene}
-          <Header options={descriptor.options} onPop={() => childNav.pop()} allowBack={index !== 0} />
+          <Header loggedIn={selectedTab} options={descriptor.options} onPop={() => childNav.pop()} allowBack={index !== 0} />
```

Comment:
`!!selectedTab`. Seems a little fragile? Maybe `nameToTab` should have a note about it.